index.html

```diff
@@ -26,7 +32,158 @@ <h1>I’m Mariane Glad to see you!</h1>
             <li><a href="#"><img src="images/Github.svg" alt="Github"></a></li>
             <li><a href="#"><img src="images/Angel List.svg" alt="Angel List"></a></li>
         </ul>
+    </section>
+    <!--Work section-->
+    <section id="work">
+
+        <!--Portfolio Card -1-->
+        <div class="portfolio-snapshot">
+            <a class=img href="#"><img src="images/portfolio-snapshot-1.svg" alt="Portfolio-Snapshot-1"></a>
+
+            <h1 class="project-title">Tonic</h1>
+            <ul class="project-name">
+                <li class="sub-title"><a href="#">CANOPY</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">Back End Dev</a></li>
+                <li class="bullet">&#x2022;</li>
+                <li><a class="backend-yr" href="#">2015</a></li>
+            </ul>
+            <p class="description">A daily section of privately personalized reads; no accounts or sign-ups
```

Comment:
- [ ] I think it's better to avoid lines of code longer than 100 characters following the best practices of coding. For more information [click here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
Thank for the information. Since this is not a project requirement, I will take this as optional. Thank you.